index/evaluation.py

```diff
@@ -177,3 +271,69 @@ def score_mappings(matches: pd.DataFrame) -> float:
     matches = matches[matches["target_concept_label"].notnull()]
     accuracy = matches["correct"].sum() / len(matches)
     return accuracy
+
+
+def evaluate(
+    datasets,
+    labels,
+    store_results=False,
+    model="gpt",
+    results_root_dir="resources/results/pd",
+):
+
+    if model == "gpt":
+        data_gpt = {}
+        data_fuzzy = {}
+        for idx, source in enumerate(datasets):
+            acc_gpt = []
+            acc_fuzzy = []
+            for idy, target in enumerate(datasets):
+                map_gpt = match_closest_descriptions(source, target)
+                map_fuzzy = match_closest_descriptions(
+                    source, target, matching_method=MatchingMethod.FUZZY_STRING_MATCHING
+                )
+                if target == "jadni":
+                    print("check")
```

Comment:
What is this check for? :D

Reply:
It was there in the original code, so I just left it :D